frontend/src/Sidebar.ts

```
@@ -384,21 +442,59 @@ export function Sidebar(sidebarHidden: Signal<boolean>) {
+                return m("span", {
+                  style: {fontStyle: "italic"},
+                },
+                NTag({
+                  type: "info",
+                  style: {width: "4em", justifyContent: "center", color: "grey", fontStyle: "italic"},
+                  size: "small",
+                }, "TABLE"),
+                option.label);
+              case "endpoint":
+                // Virtual endpoint item
+                const parts = option.key.split(":");
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)